src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -278,7 +278,7 @@ public async Task RefreshAsync(CancellationToken cancellationToken)
                             {
                                 // Trigger a single load-all operation if a change was detected in one or more key-values with refreshAll: true
                                 applicationSettings = await LoadAll(client, cancellationToken).ConfigureAwait(false);
-
+                                _logger?.LogInformation(LoggingConstants.SuccessfulConfigurationUpdated + " on endpoint: " + AppConfigurationEndpoint); // what endpoint is client coming from? for georeplication

```

Comment:
>AppConfigurationEndpoint [](http://example.com/codeflow?start=125&length=24)

This is not the current client's endpoint, its just he first endpoint in the list. We can add a new method in `IConfigurationClientManager` to return the endpoint of the current client.